contracts/chain-adapters/ForwarderBase.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+abstract contract ForwarderBase is UUPSUpgradeable {
+    // L3 address of the recipient of fallback messages and tokens.
```

Comment:
the l3 address is the recipient of fallback messages? do you mean relayed l1 to l2 messages?

Reply:
Yes. I changed the comment https://github.com/across-protocol/contracts/pull/609/commits/814498b0feb17b83088e27eaba26306cd0ac0108